sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -179,45 +205,33 @@ private void saveSymbolHighlighting(List<SymbolHighlightingData> highlightingDat
     }
   }
 
-  private void saveSyntaxHighlighting(List<SyntaxHighlightingData> highlightingDataList, InputFile inputFile) {
-    Highlightable highlightable = perspective(Highlightable.class, inputFile);
-    if (highlightable != null) {
-      HighlightingBuilder highlightingBuilder = highlightable.newHighlighting();
-
-      for (SyntaxHighlightingData highlightingData : highlightingDataList) {
-        highlightingBuilder.highlight(highlightingData.startOffset(), highlightingData.endOffset(), highlightingData.highlightCode());
-      }
-
-      highlightingBuilder.done();
-    }
-
-  }
-
   @Nullable
-  <P extends Perspective<?>> P perspective(Class<P> clazz, InputFile file) {
+  <P extends Perspective> P perspective(Class<P> clazz, InputFile file) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Perspective"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)